openassessment/xblock/grade_mixin.py

```diff
@@ -584,3 +585,80 @@ def _assessment_grade_context(self, assessment):
                     part['option']['label'] = option_labels.get(option_label_key, part['option']['name'])
 
         return assessment
+
+    def _get_score_explanation(self, assessment_steps, score):
+        """
+        Return a string which explains how grade is calculated for the ORA assessment
+        based on assessment_steps.
+        Args:
+            assessment_steps (list): List of assessment_steps for some ORA.
+            score (dict): Dict containing points_possible, points_earned and its related attributes
+        Returns:
+            str
+        """
+
+        first_sentence = _(
+            "You have successfully completed this problem and received a {earned_points}/{total_points}."
+        ).format(earned_points=score["points_earned"], total_points=score["points_possible"])
+
+        second_sentences_options = {
+            "self": _("The grade for this problem is determined by your Self Assessment."),
```

Comment:
You'll need to update the translations using the available make files.
The error can be reproduced locally running:
```
make check_translations_up_to_date
```
And to fix the issue:
```
make extract
```